app/helpers/application_helper.rb

```diff
@@ -103,16 +107,17 @@ def habitats_present_modal
     map_to_citations_string(I18n.t('countries.shared.habitats_present.citations'))
   end
 
+
   def map_to_citations_string translations
     citations = translations.map do |cit|
       "<p>#{cit}</p>"
     end
 
-    citations.join
+    "<h3>#{I18n.t('global.sources_modal_title')}</h3>" + citations.join
   end
 
   def habitat_text(habitat)
-    habitat[:change_percentage].blank? ? I18n.t('home.habitat_cover.data_deficient') : habitat[:text].html_safe
+    habitat[:change_percentage].nil? ? '' : habitat[:text].html_safe
```

Comment:
@stanleypliu This still shows the data_deficient text - if no data is present, it just won't show '-' next to the icon:
<img width="260" alt="Screenshot 2020-07-06 at 08 34 01" src="https://user-images.githubusercontent.com/43963744/86567541-72fbe600-bf63-11ea-8a57-0259cabfe55f.png">

Reply:
Ahhh, I deleted it. No worries I'll add that back in.